src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -2170,6 +2212,13 @@ std::error_code Session::receive_unbound_message()
 }
 
 
+std::error_code Session::receive_query_error_message(int error_code, std::string_view message, int64_t query_version)
+{
+    logger.info("Received QUERY_ERROR \"%1\" (error_code=%2, query_verison=%3)", message, error_code, query_version);
```

Comment:
verison->version

Reply:
fixed. this is my 2nd most frequent typo, right after `relam`.

Reply:
I'm with you there for `relam`, gotta type fast!